addons/website_mail_channel/views/snippets.xml

```diff
@@ -16,7 +16,7 @@
             <div class="oe_snippet_body js_follow_alias"
                       data-id="0"
                       data-object="mail.channel"
-                      data-follow="off">
+                      data-follow="off" data-name="Discussion Group">
```

Comment:
Maybe a great time to make an actual snippet out of this ? (a view with a dedication folder + options etc) ? So that this is not needed. (Maybe do that in a separate commit so that I can try keeping the history the day I merge it).

Reply:
Alright, I will add it on top of the current PR